### PR TITLE
Make 1.14.6 more positive per #1192

### PR DESCRIPTION
**Pull request by tghosth**
This Pull Request relates to issue #1192

### CODE REVIEW

**Comment by tghosth** [comment]
Jim already approved this here: https://github.com/OWASP/ASVS/issues/1192#issuecomment-1369269880

File: 5.0/en/0x10-V1-Architecture.md [hunk 1]
```diff
@@ -130,7 +130,7 @@ This is a placeholder for future architectural requirements.
...
 | **1.14.4** | Verify that the build pipeline contains a build step to automatically build and verify the secure deployment of the application, particularly if the application infrastructure is software defined, such as cloud environment build scripts. | | ✓ | ✓ | |
 | **1.14.5** | Verify that application deployments adequately sandbox, containerize and/or isolate at the network level to delay and deter attackers from attacking other applications, especially when they are performing sensitive or dangerous actions such as deserialization. ([C5](https://owasp.org/www-project-proactive-controls/#div-numbering)) | | ✓ | ✓ | 265 |
-| **1.14.6** | Verify the application does not use unsupported, insecure, or deprecated client-side technologies such as NSAPI plugins, Flash, Shockwave, ActiveX, Silverlight, NACL, or client-side Java applets. | | ✓ | ✓ | 477 |
+| **1.14.6** | Verify the application only uses client-side technologies which are still supported and considered secure. Examples of technologies which do not meet this requirement include NSAPI plugins, Flash, Shockwave, ActiveX, Silverlight, NACL, or client-side Java applets. | | ✓ | ✓ | 477 |
```
Comment (elarlang) [inline 5.0/en/0x10-V1-Architecture.md:133]: you know, `MODIFIED` :)

**Review by elarlang — CHANGES_REQUESTED** [review]
label

**Comment by tghosth** [comment]
> label

🤦‍♂️fixed

**Review by jmanico — APPROVED** [review]

**Comment by tghosth** [comment]
@elarlang you need to approve

**Review by elarlang — APPROVED** [review]

**Pull request merged**